### DISCUSSION TITLE
Archer T600U Nano cant connect to Wi-fi

### DISCUSSION THREAD

**Discussion by Zjivago** [chris1111/Wireless-USB-Big-Sur-Adapter#328] [Q&A]
Trying to help my daughter with her Macbook Pro 13 2019 (A2159) that has awful Wi-fi reception (faulty antennas most likely). 
I was able to install the program follow the step by step instructions, and the adapter became active. It seems to have a good signal to my network but when typing in the password (checked several times) it just tries to connect a couple of times but nothing much happens. The adapter is blinking away but no connection. Any advice? I never owned a Mac in my life so very novice.

**Comment by chris1111** [comment]
Try to make the Apple Wifi to OFF

make sur you have the right password?

**Reply by Zjivago** [reply]
Shutting down the default wifi seems to do the trick! Many thanks. The connection is a bit unstable at range but I will try to upgrade the adapter, I got the cheapest one to see if the solution worked.

**Comment by Zjivago** [comment]
> Try to make the Apple Wifi to OFF
> 
> 
> 
> make sur you have the right password?

Thank you for the reply! I havent tried to shut off the standard wifi so I will try that when I have the chance. Unless I suffer from sudden amnesia the password should be correct, I tested many many times and assured no caps.
